MAINTAINERS.md

```diff
@@ -114,6 +112,8 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   [17 January 2020](https://github.com/tldr-pages/tldr/issues/3764) — [3 February 2021](https://github.com/tldr-pages/tldr/issues/5224)
 - Axel Navarro ([@navarroaxel](https://github.com/navarroaxel)):
   [5 October 2020](https://github.com/tldr-pages/tldr/issues/4504) — [7 April 2021](https://github.com/tldr-pages/tldr/issues/5703)
+- bl-ue ([@bl-ue](https://github.com/bl-ue)):
+  [2 February 2021](https://github.com/tldr-pages/tldr/issues/5219) — [25 June 2021](https://gitter.im/tldr-pages/tldr?at=60d615598a40b117282a96d7)
```

Comment:
```suggestion
  [2 February 2021](https://github.com/tldr-pages/tldr/issues/5219) — [25 June 2021](https://github.com/tldr-pages/tldr/pull/6388/)
```

Reply:
I don't really want to use that link because it doesn't reflect _when_ I actually discontinued. I pointed to my Gitter message because that _was_ when I stopped.  That said, if you really think I should point to this PR instead, I'll do it, if a few others offer their opinions.

Reply:
Even tough we use the PR for all the others I think it makes more sense to use the Gitter message here.